jabgui/src/main/java/org/jabref/gui/JabRefGUI.java

```diff
@@ -144,6 +144,9 @@ public void start(Stage stage) {
     }
 
     public void initialize() {
+        // Set JavaFX ListView cell count limit to handle large libraries
+        System.setProperty("javafx.scene.control.ListView.maxCellCount", "50000");
```

Comment:
Is this even a real thing?
Please provide some source, some deep link to JavaFX source where this property is read, or some documentation?

Or is this some hallucination of an ai?